transcom/mymove
# Amw b 17956 total sit days used to not exceed approved days

## [Agility ticket](tbd)

## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Office side as a TOO
2. Access a move with SIT items on it
3. Navigate to the MTO tab
4. Edit the SIT details and observer that the values change to support the newly entered values in realtime.
5. Save the SIT details and observe the data saved correctly.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
